Gopkg.toml

```diff
@@ -30,7 +30,7 @@ required = [
 
 [[constraint]]
   name = "k8s.io/code-generator"
-  branch = "release-1.9"
+  branch = "release-1.10"
```

Comment:
Do we also need to update the `k8s.io/apimachinery` and `k8s.io/api` dependencies?

Reply:
All of these could've been changed. But eventually they are dependent on github.com/argoproj/argo which is still at `release-1.9`. That cause the code generation to fail if we bump up the version of `k8s.io/apimachinery` and `k8s.io/api`